### ISSUE TITLE
How to add custom keyboard

### ISSUE THREAD

**Issue by MohammadMDSA** [Microsoft/BotBuilder#3751]
## Bot Info 
* SDK Platform: Node.js
* SDK Version: 3.11.0
* Active Channels: Telegram
## Issue Description
How can I put a custom keyboard on my Telegram bot with NodeJS SDK

**Comment by FranciscoPonceGomez** [comment]
HI @MohammadMDSA,

Please add this question to your other thread. Rephrase it to a more global subject like "Questions about Bot Framework with Node.js SDK and Telegram"

Thanks,
Francisco
